src/System.Security.Permissions/ref/System.Security.Permissions.csproj

```diff
@@ -6,6 +6,8 @@
   </PropertyGroup>
   <ItemGroup>
     <Compile Include="System.Security.Permissions.cs" />
+    <Compile Include="System.Security.Permissions.$(TargetGroup).cs" />
```

Comment:
This is causing it to try to find a file System.Security.Permissions.netfx.cs, which does not exist. Maybe you need to conditionalize this to `'$(TargetsNetFx)' != 'true'` ? 

Good idea to `build -buildallconfigurations` to validate this kind of change locally (when you have changes that affect specific configurations)